### PR TITLE
DEV-43 added cbd cleanup command

### PR DESCRIPTION
**Pull request by oleewere**
@lalyos

### CODE REVIEW

File: include/compose.bash [hunk 1]
```diff
@@ -45,6 +45,41 @@ compose-kill() {
     dockerCompose rm -f
 }
 
+compose-cleanup() {
+    declare desc="Removes all exited containers and old cloudbreak related images"
+
+    if [ ! -f docker-compose.yml ];then
+      error "docker-compose.yml file does not exists"
+      return
```
Comment (lalyos) [inline include/compose.bash]: after error use exit

File: include/compose.bash [hunk 2]
```diff
@@ -45,6 +45,41 @@ compose-kill() {
...
+    fi
+
+    local exited_containers=$(docker ps --all -q -f status=exited)
```
Comment (lalyos) [inline include/compose.bash]: it could go into a separate function, so it can be reused

File: include/compose.bash [hunk 3]
```diff
@@ -45,6 +45,41 @@ compose-kill() {
...
+        fi;
+        for version in ${image_versions%?}
+        do
+          if [ "$version" != "$image_version" ]; then
+            info "Found different image version for $image_name: $version ($image_version), start deleting.."
+            docker rmi "$image_name:$version"
+          fi
+        done
+    done
+}
```
Comment (lalyos) [inline include/compose.bash]: please use functions to filter images
```
img=$(docker images|sed "s/ \+/ /g"|cut -d' ' -f 1,2|tr ' ' : | sed -n 'H; $ {x;s/\n/#/g;p;}')
keep=....

func filterImagesToDelete() {
  declare all=$1 keep=$2
  ...
  print <list of images to delete.
```

**Comment by oleewere** [comment]
@lalyos : updated

**Pull request closed without merging**